README.rdoc

```diff
@@ -1,10 +1,10 @@
 == README
 
-##### Introduction
+#### Introduction
 TexArch is an online tool hosting data supportive to archaeological research of Sergio Ayala at Gault School of Archaeological Research in San Marcos, TX.
 The application introduces to the research, displays artifacts data and images and shows sites on a dynamic google map.
 
```

Comment:
TexArch is an online tool hosting data in support of the research of Sergio Ayala at the Gault School of Archaeological Research in San Marcos, TX. The application introduces the research and displays artifacts' data and images on a dynamic Google map.